pandas/core/indexes/datetimelike.py

```diff
@@ -430,10 +430,7 @@ def min(self, axis=None, *args, **kwargs):
         --------
         numpy.ndarray.min
         """
-        if axis is not None and axis >= self.ndim:
-            raise ValueError("`axis` must be fewer than the number of "
-                             "dimensions ({ndim})".format(ndim=self.ndim))
-
+        _validate_minmax_axis(axis)
```

Comment:
not what i mean
add this specifically to no.validate_* there are mechanisms for this already

Reply:
I see, done.